docs/reference/supported-environments.md

```diff
@@ -117,28 +120,51 @@ Camunda 8 works with the [default distribution](https://www.elastic.co/downloads
-| -                     | -           | -                                                                          | -               |
-| Web Modeler 8.7.x     | Zeebe 8.7.x | Operate 8.7.x Tasklist 8.7.x Identity 8.7.x Connectors 8.7.x Console 8.7.x | Optimize 8.7.x  |
-| Web Modeler 8.6.x     | Zeebe 8.6.x | Operate 8.6.x Tasklist 8.6.x Identity 8.6.x Connectors 8.6.x Console 8.6.x | Optimize 8.6.x  |
-| Web Modeler 8.5.x     | Zeebe 8.5.x | Operate 8.5.x Tasklist 8.5.x Identity 8.5.x Connectors 8.5.x Console 8.5.x | Optimize 8.5.x  |
-| Web Modeler 8.4.x     | Zeebe 8.4.x | Operate 8.4.x Tasklist 8.4.x Identity 8.4.x Connectors 8.4.x               | Optimize 8.4.x  |
-| Web Modeler 8.3.x     | Zeebe 8.3.x | Operate 8.3.x Tasklist 8.3.x Identity 8.3.x Connectors 8.3.x               | Optimize 8.3.x  |
-| Web Modeler 8.2.x     | Zeebe 8.2.x | Operate 8.2.x Tasklist 8.2.x Identity 8.2.x Connectors 0.23.2              | Optimize 3.10.x |
+The following matrix shows which component versions work together.
+
+<Tabs
+defaultValue="version-8-7-x"
+values={[
+{label: 'Version <= 8.7.x', value: 'version-8-7-x'},
+{label: 'Version 8.8.x+', value: 'version-8-8-x'},
+]}>
+
+<TabItem value="version-8-7-x">
+
+- Please note that `Zeebe`, `Operate`, and `Tasklist` form the Core Orchestration cluster and must be running on the exact same `minor`, and `patch` level starting from version `8.6.0` going forward.
+
+| Design                                        | Automate                                                                                         | Improve         |
+| --------------------------------------------- | ------------------------------------------------------------------------------------------------ | --------------- |
+| Desktop Modeler 5.31+ <br/> Web Modeler 8.7.x | (Zeebe, Operate, Tasklist) 8.7.x, Connectors 8.7.x, <br/>Identity 8.7.x, Console 8.7.x           | Optimize 8.7.x  |
+| Desktop Modeler 5.28+ <br/> Web Modeler 8.6.x | (Zeebe, Operate, Tasklist) 8.6.x, Connectors 8.6.x, <br/>Identity 8.6.x, Console 8.6.x           | Optimize 8.6.x  |
+| Desktop Modeler 5.22+ <br/> Web Modeler 8.5.x | Zeebe 8.5.x, Operate 8.5.x, Tasklist 8.5.x, <br/>Identity 8.5.x, Connectors 8.5.x, Console 8.5.x | Optimize 8.5.x  |
+| Desktop Modeler 5.19+ <br/> Web Modeler 8.4.x | Zeebe 8.4.x, Operate 8.4.x, Tasklist 8.4.x, <br/>Identity 8.4.x, Connectors 8.4.x                | Optimize 8.4.x  |
+| Desktop Modeler 5.16+ <br/> Web Modeler 8.3.x | Zeebe 8.3.x, Operate 8.3.x, Tasklist 8.3.x, <br/>Identity 8.3.x, Connectors 8.3.x                | Optimize 8.3.x  |
+| Desktop Modeler 5.10+ <br/> Web Modeler 8.2.x | Zeebe 8.2.x, Operate 8.2.x, Tasklist 8.2.x, <br/>Identity 8.2.x, Connectors 0.23.2               | Optimize 3.10.x |
+| Desktop Modeler 4.12+                         | Zeebe 1.3.x, Operate 1.3.x, Tasklist 1.3.x, IAM 1.3.x                                            | Optimize 3.7.x  |
```

Comment:
@RomanJRW can we remove Optimize 3.7 from this table?

Reply:
I removed it because we got rid of that row (given that we do not support that version anymore anyhow.)